includes/Data/SiteFeatures.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace NewfoldLabs\WP\Module\Onboarding\Data;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+
+final class SiteFeatures {
+
+	protected static $default_plugins = array(
+		array(
+			'slug'     => 'jetpack',
+			'icon'     => '--site-features-security',
+			'title'    => 'Security, Speed & Growth',
+			'subtitle' => 'Powered by Jetpack',
+			'desc'     => 'Jetpack',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'wpforms-lite',
+			'icon'     => '--site-features-form',
+			'title'    => 'Forms',
+			'subtitle' => 'Powered by WP Forms',
+			'desc'     => 'WP Forms',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'google-analytics-for-wordpress',
+			'icon'     => '--site-features-analytics',
+			'title'    => 'Site Traffic',
+			'subtitle' => 'Powered by MonsterInsights',
+			'desc'     => 'MonsterInsights',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'wordpress-seo',
+			'icon'     => '--site-features-share',
+			'title'    => 'Search Engine Optimization',
+			'subtitle' => 'Powered by Yoast',
+			'desc'     => 'Yoast',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'creative-mail-by-constant-contact',
+			'icon'     => '--site-features-email',
+			'title'    => 'Email Newsletters',
+			'subtitle' => 'Powered by Creative Email',
+			'desc'     => 'Creative Email',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'yith-woocommerce-ajax-search',
+			'icon'     => '--site-features-search',
+			'title'    => 'Enhanced Product Search',
+			'subtitle' => 'Powered by YITH',
+			'desc'     => 'YITH',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'nfd_slug_yith_woocommerce_ajax_product_filter',
+			'icon'     => '--site-features-filter',
+			'title'    => 'Enhanced Product Filters',
+			'subtitle' => 'Powered by YITH',
+			'desc'     => 'YITH',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'nfd_slug_yith_woocommerce_booking',
+			'icon'     => '--site-features-bookingcalendar',
+			'title'    => 'Bookings & Appointments',
+			'subtitle' => 'Powered by YITH',
+			'desc'     => 'YITH',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'nfd_slug_yith_woocommerce_wishlist',
+			'icon'     => '--site-features-wishlist',
+			'title'    => 'Product Wishlists',
+			'subtitle' => 'Powered by YITH',
+			'desc'     => 'YITH',
+			'selected' => false,
+		),
+		array(
+			'slug'     => 'optinmonster',
+			'icon'     => '--site-features-lead',
+			'title'    => 'Lead Generation',
+			'subtitle' => 'Powered by Optin Monster',
+			'desc'     => 'Optin Monster',
+			'selected' => false,
+		),
+	);
+
+	public static function convert_to_hash() {
+		$idx                  = 0;
+		$default_plugins_hash = array();
+
+		foreach ( self::$default_plugins as $default_plugin ) {
+			$default_plugins_hash[ $default_plugin['slug'] ] = $idx;
+			$idx += 1;
+		}
+
+		return $default_plugins_hash;
+	}
```

Comment:
Yeah I was initially doing that but then the front end would have to be expecting a key value Object rather than a array and then the further use of that component will require this exact signature.

Reply:
Also, we won't be able to use **map** and all further in our Components